code/game/machinery/newscaster.dm

```diff
@@ -200,6 +201,8 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 				<BR><A href='?src=\ref[src];view=1'>View Feed Channels</A>
 				<BR><A href='?src=\ref[src];create_feed_story=1'>Submit new Feed story</A>
 				<BR><A href='?src=\ref[src];menu_paper=1'>Print newspaper</A>
+				<BR><A href='?src=\ref[src];silence=0'>Silence unit</A>
```

Comment:
And now they both do, with no changes. How can I make them trade out like I wanted to?